subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/problems/KnownProblemIds.groovy

```diff
@@ -80,6 +80,9 @@ class KnownProblemIds {
         // categories from integration tests
         'TEST_PROBLEM',
         'generic:deprecation:plugin',
+        'generic:configuration-consumption-detachedconfiguration',
+        'generic:behaviour-using-task-validateplugins-without-applying-the-java-toolchain-plugin',
+        'generic:method-repositoryhandler-jcenter',
```

Comment:
🤔 I think we should now assert the deprecation problem in addition to the deprecation when we test the deprecation. Again, maybe not in this PR, but very soon.

Reply:
We plan to do that separately.
Already in the backlog.